boards/arm/lpcxpresso55s16/Kconfig.defconfig

```diff
@@ -0,0 +1,35 @@
+# LPCXpresso55S16 board
+
+# Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_LPCXPRESSO55S16
+
+config BOARD
+	default "lpcxpresso55S16"
+
+config UART_MCUX_FLEXCOMM_0
+	default y if UART_CONSOLE
+	depends on UART_MCUX_FLEXCOMM
+
+if PINMUX_MCUX_LPC
+
+config PINMUX_MCUX_LPC_PORT0
```

Comment:
I think these port instances are going away. Need to sync with @galak

Reply:
They will be going away, but for now they are still in use.